src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -126,22 +129,25 @@ public async override Task RunAsync()
 
                 await syncAction.RunAsync();
             }
-            else
-            {
-                ColoredConsole.Error.WriteLine(ErrorColor(Constants.Errors.ExtensionsNeedDotnet));
-            }
         }
 
         private async Task AddPackage(string extensionsProj, string pacakgeName, string version)
         {
-            var args = $"add \"{extensionsProj}\" package {pacakgeName} --version {version}";
-            if (!string.IsNullOrEmpty(Source))
+            if (CommandChecker.CommandExists("dotnet"))
```

Comment:
This should be done with all the other validations. No need to check for dotnet before adding every single package.

Reply:
I agree. This probably adds an overhead. Will update.